### PR TITLE
Fix recurring error in event syntax boxes

### PR DESCRIPTION
**Pull request by Elchi3**
#### Summary
@ddbeck noted there is a recurring syntax error in the new event page syntax boxes. 

I think we could omit the braces also and have an easier life / more readable syntax boxes.

#### Motivation

It gets copy pasted to more new-style event pages and we should stop this plague.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/xrsession/select_event/index.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ The WebXR **`select`** event is sent to an {{domxref("XRSession")}} when one of
 Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
 
 ```js
-addEventListener('select', (event) => { });
+addEventListener('select', event => { };
```
[the remark this answers is not in the document]
Comment (ddbeck) [reply files/en-us/web/api/xrsession/select_event/index.md]: Someday there will be a vaccine (Prettier) to prevent these.

**Review by wbamberg — COMMENTED** [review]
I don't know if it's worth fixing in this PR, but:

* do we have an opinion on single versus double quotes? We're inconsistent here. I guess single?
* in the JS docs we omit semicolons in the syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#syntax - do we want to do the same here?

**Comment by Elchi3** [comment]
I think it is a good idea.
I've used single quotes throughout and removed the semicolons

**Review by wbamberg — APPROVED** [review]
👍 thanks Florian!

**Pull request merged**